peachjam_api/public_views.py

```diff
@@ -120,3 +160,31 @@ class RatificationsViewSet(viewsets.ReadOnlyModelViewSet):
         "updated_at": ["exact", "gte", "lte"],
     }
     ordering = ["updated_at"]
+
+
+class ValidateExpressionFrbrUrisView(APIView):
+    """Validates a list of decision expression FRBR URIs and returns those that aren't valid."""
+
+    permission_classes = (IsAuthenticated,)
```

Comment:
Is there an additional permission needed? Perhaps any of the per-content-type permissions?

Reply:
hmm, i don't think we need any additional perms for this view, its a readonly view that returns a list of frbr uris